R/Tendril.plot.R

```diff
@@ -44,19 +44,32 @@
 #' print(res)
 #' @export
 
-plot.Tendril <- function(x, type = c("basic","permutations","percentile")[1], coloring = "Terms", ...){
+plot.Tendril <- function(
+  x,
+  type = c("basic","permutations","percentile")[1],
+  coloring = "Terms",
+  engine = c("ggplot2", "plotly")[1],
```

Comment:
Isn't it better to add "interactive" as a new option for the type argument? A "basic" plot will then be directed to ggplot2 plotting and "interactive" will be directed towards plotly.
I think the "permutations" and "percentile" options should be moved to a separate permutation plot function, operating on the tendril.perm class, if we decided to create such a class?

Reply:
In that case, It's better to have a single boolean flag interactive = TRUE.

Reply:
The interactivity is not really needed in the permutation plots.